docs/attributes-registry/messaging.md

```diff
@@ -1,42 +1,56 @@
+| `messaging.destination.template`                     | string   | Low cardinality representation of the messaging destination name [3]                                                                                                                                                                       | `/customers/{customerId}`          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `messaging.destination.temporary`                    | boolean  | A boolean that is true if the message destination is temporary and might not exist anymore after messages are processed.                                                                                                                   |                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `messaging.destination_publish.anonymous`            | boolean  | A boolean that is true if the publish message destination is anonymous (could be unnamed or have auto-generated name).                                                                                                                     |                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `messaging.destination_publish.name`                 | string   | The name of the original destination the message was published to [4]                                                                                                                                                                      | `MyQueue`; `MyTopic`               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `messaging.eventhubs.consumer.group`                 | string   | The name of the consumer group the event consumer is associated with.                                                                                                                                                                      | `indexer`                          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `messaging.eventhubs.message.enqueued_time`          | int      | The UTC epoch seconds at which the message has been accepted and stored in the entity.                                                                                                                                                     | `1701393730`                       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `messaging.gcp_pubsub.message.ordering_key`          | string   | The ordering key for a given message. If the attribute is not present, the message does not have an ordering key.                                                                                                                          | `ordering_key`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
let me break it down into individual groups along with the DB attributes

Reply:
Merged in your changes.